app/extensions.js

```diff
@@ -137,6 +137,12 @@ let generateBraveManifest = () => {
     cspDirectives['style-src'] = '\'self\' \'unsafe-inline\' http://' + devServer
   }
 
+  // TODO:
+  // * Move WebTorrent to its own renderer process, similar to the way it's done in
+  //   WebTorrent Desktop
+  // * Remove this CSP exception:
+  cspDirectives['connect-src'] = '*'
```

Comment:
@bbondy do these CSP rules apply only to the braveExtension, or to *all* extensions?